src/components/NavBar/NavBar.jsx

```diff
@@ -74,7 +78,25 @@ const NavBar = ({ activeProduct, user, helpMenuItems }) => (
       <Select
         hideSearch
         customButton={handleClick => (<NavBarMenu user={user} onClick={handleClick} />)}
-        items={user.menuItems}
+        items={[
+        {
+          id: 'Account',
+          title: 'Account',
+          icon: <PersonIcon color={gray} />,
+          onItemClick: () => { window.open(`https://account.buffer.com?product=${activeProduct}&username=${encodeURI(user.name)}`); },
```

Comment:
According to Mia's latest update, I believe we should be opening the Account app in the same tab 🤔  cc @ssvmvss

Reply:
thanks for the heads-up @ivanaszuber, make sense, will make the change